WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TitleAndTextFieldRow.swift

```diff
@@ -0,0 +1,56 @@
+import SwiftUI
+
+struct TitleAndTextFieldRow: View {
+    let title: String
+    let placeholder: String
+    @State var text: String = ""
+    let symbol: String?
+    let keyboardType: UIKeyboardType
+
+    var body: some View {
+        HStack {
+            Text(title)
+                .font(.body)
+                .lineLimit(1)
+                .fixedSize()
+            Spacer()
+            TextField(placeholder, text: $text)
+                .multilineTextAlignment(.trailing)
+                .font(.body)
+                .keyboardType(keyboardType)
+            if let symbol = symbol {
+                Text(symbol).font(.body)
+            }
+        }
+        .frame(height: 44)
+        .padding([.leading, .trailing], 16)
```

Comment:
Same question about having these magic numbers in enums. Just pointing them out in case we decide to change it, we don't miss them.